archivesunleashed/auk
# Use http instead of DOM to access gexf data in Sigma -- Issue 194

**GitHub issue(s)**:

#194, also related to #189 and #191.

# What does this Pull Request do?

Instead of having rails process the data and storing in the DOM for more processing, this script simply collects the url from Ruby and then accesses the gexf data via http.

# How should this be tested?

For now, the sigmajs graphs should work exactly the same as before. A nice bonus would be that we would have a time-speed up for large graphs, but that's not a required test for this (I will see if we can leverage this change to fix #189 and/or #19.

# Additional Notes:


# Interested parties

Tag (@ mention) interested parties.

Thanks in advance for your help with the Archives Unleashed Toolkit!
